### DISCUSSION TITLE
Longer "Top priorities" list

### DISCUSSION THREAD

**Discussion by thecserko** [kudrykv/latex-yearly-planner#84] [General]
Hello,

I just found this excellent planner last week and using it actively since on my Note Air 2 with the monthly calendar on the daily view. Is there perhaps a version already, or could it be made, where on the daily view the "Top priorites" to do list is a little bit longer? Like 12 - 14 item long (it is 8 item long now).

Thanks!

**Comment by kudrykv** [comment]
Hi @thecserko!

Yep, this is configurable. What planner variation do you use?

**Reply by thecserko** [reply]
Hello @kudrykv,

I believe i am using the "Annual planner with months on side for 2023, 2024", DDVK, right hand, w/ daily cal, 2023. But maybe i should use the non-ddvk remarkable version? I am a little lost on it... I have a boox note air 2.

Anyway, if you could configure it with the little longer top priorities list, it would be awesome. Since i wrote, i also noticed that some versions have dedicated to do lists, that is accessible from the daily views like the notes section. Mine is missing that. Could you include it too?

Thank you, i highly appreciate your kind help!

thecserko

**Reply by kudrykv** [reply]
The version with a dedicated to-do list is currently unavailable 😬

Those templates are all the same. The difference is with the shift of the whole page to account native devices menu bars. If you didn't see any issues with DDVK, then DDVK it is 🙂

[rm2_ddvk.mos.dotted.default.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11819052/rm2_ddvk.mos.dotted.default.dailycal.2023.pdf)
[rm2_ddvk.mos.dotted.default.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11819053/rm2_ddvk.mos.dotted.default.dailycal.2024.pdf)

Please check

**Reply by thecserko** [reply]
Dear @kudrykv!

Awesome! Thanks! Only i forgot to tell you, that i use the lined version, not the dotted :(. Sorry for the additional work!

Thank you!

**Reply by kudrykv** [reply]
I hope it's still 24 hrs and Monday start 🙂

[rm2_ddvk.mos.lined.default.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11824308/rm2_ddvk.mos.lined.default.dailycal.2023.pdf)
[rm2_ddvk.mos.lined.default.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11824309/rm2_ddvk.mos.lined.default.dailycal.2024.pdf)

**Reply by thecserko** [reply]
Man, that was fast, like superfast 😃 .

Thank you, it is perfect now!

May i ask, why is the to do version unavailable? Is it some kind of programming issue?

**Reply by kudrykv** [reply]
I killed it some time ago, and now it's too cumbersome to go back; there were a bunch of fixes on top of that, so reverting to that version would also revert some bugs that were on that time (some wrong navigation primarily).

The "refactor" with putting it back is moving, but it is slow 😔
